cypress/integration/types/constants.ts

```diff
@@ -1,23 +1,22 @@
+export const businessservices = "Business services";
+export const button = "button";
+export const clearAllFilters = "Clear all filters";
 export const controls = "Controls";
+export const createNewButton = "Create new";
+export const displayNameFilter = "Display name";
+export const emailFilter = "Email";
+export const groupFilter = "Group";
+export const invalidEmailMsg = "This field requires a valid email.";
 export const jobfunctions = "Job functions";
+export const jobfunctionFilter = "Job function";
+export const nameFilter = "Name";
 export const stakeholders = "Stakeholders";
 export const stakeholdergroups = "Stakeholder groups";
-export const businessservices = "Business services";
-export const trTag = "tr";
 export const tdTag = "td";
-export const createNewButton = "Create new";
-export const button = "button";
-export const emailFilter = "Email";
-export const displayNameFilter = "Display name";
-export const jobfunctionFilter = "Job function";
-export const groupFilter = "Group";
-export const clearAllFilters = "Clear all filters";
+export const trTag = "tr";
 
 // Error helper messages general to all views
-export const minCharsMsg = "This field must contain at least 3 characters.";
+export const duplicateErrMsg = "ERROR: duplicate key value violates unique constraint";
 export const max120CharsMsg = "This field must contain fewer than 120 characters.";
 export const max250CharsMsg = "This field must contain fewer than 250 characters.";
-export const duplicateErrMsg = "ERROR: duplicate key value violates unique constraint";
-
-// Stakeholder specific helper/error messages
-export const invalidEmailMsg = "This field requires a valid email.";
+export const minCharsMsg = "This field must contain at least 3 characters.";
```

Comment:
where did you move `invalidEmailMsg`?

Reply:
To line 9 , just sorted it

Reply:
I think, @nitishSr has differentiated that because only stakeholders have that field.
It will be better if we modify this file by differentiating values in various sections(by comment)
e.g.  `// Stakeholder specific helper/error messages` , `//Entity tab names` etc

Reply:
@sshveta what's your opinion about it?

Reply:
@ganeshhubale  : even if it is stakeholder specific , it is part of constants now . If you see there are other constants that are specific to job functions . so this is fine .